frontend/src/api/Error.ts

```diff
@@ -0,0 +1,30 @@
+export type ErrorResponse = {
+  error: boolean,
+  status: string,
+  message: string;
+};
+
+type AxiosError = {
+  response?: {
+    status: string,
+    data: any,
+  },
+};
+
+export const errorHandler = (err: AxiosError): ErrorResponse => {
+  if (!err.response) {
+    return {
+      error: true,
+      status: 'XXX',
```

Comment:
Does this `XXX` value hold any meaning -- if initiating the REST request failed, is there any specific error status associated with that? If not, no problem.

Reply:
I think it's when the request doesn't initiate in the first place so no status nor data is returned, hence the 'XXX' as a dummy value. I added a comment for that but can also replace it w/ something else like '000' or '0' or make it an optional param; open to ideas.

Reply:
I think that's fine, 'XXX' as a dummy value makes sense.